code/src/components/Square.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+import { useSelector } from "react-redux";
+
+export const Square = ({ coordinates }) => {
+  const gamePosition = useSelector(
+    store => store.game.currentGameState.gameData.coordinates
+  );
+  const coordinatesArray = useSelector(store => store.game.coordinatesArray);
+  const visitedSteps = coordinatesArray.find(item => item === coordinates);
+
+  return (
+    <MapSquare
+      coordinates={coordinates}
+      gamePosition={gamePosition}
+      coordinatesArray={coordinatesArray}
+      visitedSteps={visitedSteps}
+    >
+      <MapSquareP coordinates={coordinates} gamePosition={gamePosition}>
+        {coordinates}{" "}
+        {coordinates === gamePosition && gamePosition === "1,3" ? (
+          <Span>Finish</Span>
+        ) : coordinates === gamePosition && gamePosition === "0,0" ? (
+          <Span>Start</Span>
+        ) : (
+          ""
```

Comment:
Don't think you need this, since if the condition specified is not met then no text will be printed.